core/src/com/unciv/logic/civilization/PolicyManager.kt

```diff
@@ -24,8 +24,15 @@ class PolicyManager {
     var numberOfAdoptedPolicies = 0
     var shouldOpenPolicyPicker = false
         get() = field && canAdoptPolicy()
-    var legalismState = HashMap<String, String>()
+
+    private var cultureBuildingsAdded = HashMap<String, String>() // Maps cities to buildings
+    private var specificBuildingsAdded = HashMap<String, MutableSet<String>>() // Maps buildings to cities
+    private var maintenanceFreeBuildings = HashMap<String, MutableSet<String>>() // Maps cities to buildings
     var autocracyCompletedTurns = 0
+    
+    @Deprecated("Deprecated since 3.14.16") // Replaced with cultureBuildingsAdded
+    var legalismState = cultureBuildingsAdded // Maps cities to buildings
+    // We make it a reference copy of the original variable. This way, it can still works in older versions
```

Comment:
The problem is that if I have an old saved game, and I upgrade, I lose my older data. The way we've dealt with these kinds of things so far is to have a 'copying' function in GameInfo.setTransients from old to new variables.